### DISCUSSION TITLE
Animatediff bug?

### DISCUSSION THREAD

**Discussion by Nick-xl** [continue-revolution/sd-webui-animatediff#519] [Q&A]
<img width="898" alt="Snipaste_2024-05-11_00-31-15" src="https://github.com/continue-revolution/sd-webui-animatediff/assets/168004358/0b8bbde9-5bdf-430f-a5d6-45a6d21270ff">
<img width="884" alt="Snipaste_2024-05-11_00-31-25" src="https://github.com/continue-revolution/sd-webui-animatediff/assets/168004358/4db90d34-3aa2-4d0f-988d-17acd21614bb">

Where is the animation stuck? I would like to ask the gods of the discussion board, if it is solved.
torch 2.3,CUDA12.1,

**Comment by xb191799981** [comment]
![903CA1C0-1159-41bc-A584-E7247884ED63](https://github.com/continue-revolution/sd-webui-animatediff/assets/6196072/09ab1ef0-8b18-47c0-bcfc-712d4a21e110)
我也碰到同样的问题了，总是提示下标越界。
